README.md

```diff
@@ -28,6 +28,11 @@ See the official [React Native website](https://facebook.github.io/react-native/
 ## Getting Started
 
 - Follow the [Getting Started guide](http://facebook.github.io/react-native/docs/getting-started.html) to install React Native and its dependencies.
+- Install the [Chocolatey package manager](https://chocolatey.org/) in an Administrator command prompt or PowerShell
+  * In the same Administrator command prompt, type ``choco install nodejs.install``
```

Comment:
I'd rather put this in a separate section (e.g., Chocolatey installation instructions)